lib/bowling_scorecard.rb

```diff
@@ -0,0 +1,38 @@
+
+class BowlingScorecard
+  def initialize(rolls)
+    @rolls = rolls
+  end
+  def set_frames
+    @rolls.map{@rolls[0] == 10 ? @rolls << @rolls.shift(1) : @rolls << @rolls.shift(2)}
```

Comment:
conventional if statement might read better than the ternary operator in this case